applications/ecallmgr/src/ecallmgr_fs_authz.erl

```diff
@@ -31,9 +31,16 @@
 
 -export_type([authz_reply/0]).
 
--spec authorize(kzd_freeswitch:data(), ne_binary(), atom()) -> authz_reply().
+-spec authorize(kz_evt_freeswitch:data(), ne_binary(), atom()) -> authz_reply().
```

Comment:
This is called from `ecallmgr_fs_router_util:authorize_call_fun/5` and passes in a `kz_proplist()`. I suspect this will fail when processed by ecallmgr_fs_authz?

Reply:
`ecallmgr_fs_router_util:authorize_call_fun/5` no longer used, removing it